README.md

```diff
@@ -1,7 +1,15 @@
 # pledge
-a slack bot that helps your team keep commitments
 
-# how to use
+A slack bot that helps your team keep commitments
+
+# quick start
+
+## install
+
+- clone this repo, then `npm run build`, `npm run start`
+- with your browser, go to `http://my.pledge.deploy.com/add-to-slack` and authorize pledge on your slack team
```

Comment:
I stumbled upon the URL for a few seconds wondering what magical service `http://deploy.com/` was.

Can you change this to `http://pledge.example.com/add-to-slack`?